aries_cloudagent/revocation/indy.py

```diff
@@ -107,23 +110,57 @@ async def init_issuer_registry(
 
     async def handle_full_registry(self, revoc_reg_id: str):
         """Update the registry status and start the next registry generation."""
+        await self._set_registry_status_registry(
+            revoc_reg_id, IssuerRevRegRecord.STATE_FULL
+        )
+
+    async def decommission_registry(self, cred_def_id: str):
+        """Decommission active registries and start the next registry generation."""
+        async with self._profile.session() as session:
+            active = sorted(
+                await IssuerRevRegRecord.query_by_cred_def_id(
+                    session, cred_def_id, IssuerRevRegRecord.STATE_ACTIVE
```

Comment:
I would have to dig a bit to back this up but I believe there's only one "active" registry at a time and then the hot spare is in state "posted" and switched to active when the previous active is full. What I think that means is that all registries of either active or posted state would need to be retrieved and decommissioned.

Reply:
I wrote this comment before seeing the others above :sweat_smile: I'll do that digging to confirm